Package.swift

```diff
@@ -28,7 +28,9 @@ let package = Package(
             name: "TSCTestSupport",
             targets: ["TSCTestSupport"]),
     ],
-    dependencies: [],
+    dependencies: [
+        .package(url: "https://github.com/apple/swift-system.git", .upToNextMinor(from: "0.0.1"))
```

Comment:
Should this be `0.0.1` or `0.0.2`?

Reply:
It should be `0.0.3` given that the first two releases actually not compiles (`0.0.2` only compiles with release config).

`upToNextMinor()` indicates the latest `0.0.x`, once `0.0.3` is released it should resolve.